bundles/org.openhab.binding.nobohub/src/main/java/org/openhab/binding/nobohub/internal/connection/HubCommunicationThread.java

```diff
@@ -29,17 +32,49 @@
 @NonNullByDefault
 public class HubCommunicationThread extends Thread {
 
+    private enum HubCommunicationThreadState {
+        STARTING(ThingStatus.INITIALIZING, ThingStatusDetail.BRIDGE_UNINITIALIZED, ""),
```

Comment:
This will be set by the framework when `initialize()` is called, so it should not be set manually.